src/core/Alert/Alert/Alert.test.tsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+import { Alert } from './Alert';
+import { axeTest } from '../../../utils/test';
+
+describe('children', () => {
+  const alertWithElementChild = (
+    <Alert closeText="Close">
+      <div>Test alert</div>
+    </Alert>
+  );
+
+  it('has the given content', () => {
+    const { container } = render(alertWithElementChild);
+    expect(container.firstChild).toHaveTextContent('Test alert');
+  });
+
+  it('should match snapshot', () => {
+    const { container } = render(alertWithElementChild);
+    expect(container).toMatchSnapshot();
+  });
+});
```

Comment:
This could also be moved inside the `props` describe block as the `children` is one of the props we are having for the Alert.